src/models/states/lockunlock.ts

```diff
@@ -0,0 +1,5 @@
+import { State } from './state';
+
+export interface LockUnlockState extends State {
+    on: boolean;
```

Comment:
From the documentation, it seems the states are different:
https://developers.google.com/assistant/smarthome/traits/lockunlock#device-states